src/preact/base-element.js

```diff
@@ -892,7 +909,8 @@ function shouldMutationBeRerendered(Ctor, m) {
       const def = /** @type {!AmpElementPropDef} */ (props[name]);
       if (
         m.attributeName == def.attr ||
-        (def.attrs && def.attrs.includes(devAssert(m.attributeName)))
+        (def.attrs && def.attrs.includes(devAssert(m.attributeName))) ||
+        m.attributeName.startsWith(def.attrPrefix)
```

Comment:
Ditto: `startsWith` from `string.js`. And the same question is what should we do about `attrib.name === def.attrPrefix`.